testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/group/GroupMappersTest.java

```diff
@@ -109,6 +109,7 @@ private ClientRepresentation getClientByAlias(RealmRepresentation testRealmRep,
     @Test
     @SuppressWarnings("unchecked")
     public void testGroupMappers() throws Exception {
+        events.clear();
```

Comment:
This shouldn't be necessary, nor does it have anything to do with migrating LoginTests. Please keep PRs clean.